python/beeai_framework/tools/weather/openmeteo.py

```diff
@@ -46,7 +46,7 @@ class OpenMeteoToolInput(BaseModel):
     )
 
 
-class OpenMeteoTool(Tool[OpenMeteoToolInput, ToolRunOptions]):
+class OpenMeteoTool(Tool[OpenMeteoToolInput, ToolRunOptions, StringToolOutput]):
```

Comment:
now that we have JSONToolOutput isnt the output from OpenMeteo more accurately a JSONToolOutput instead of StringToolOutput?